### PR TITLE
Update account info/map

### PR DESCRIPTION
**Pull request by jfallon432**
Added update account feature. Made fixes to bugs in the JWT parsing and filter logic.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,25 @@
+<<<<<<< HEAD
```
Comment (swevelyne) [inline Dockerfile]: Please remove the merge headers

**Review by swevelyne — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (jfallon432) [reply Dockerfile]: Fixed the docker file

File: Dockerfile [hunk 2]
```diff
@@ -17,4 +18,34 @@ COPY ${JAR_FILE} app.jar
 WORKDIR /home/docker/data
 #RUN chmod +x /app.jar
 EXPOSE 5000
-ENTRYPOINT ["java", "-jar", "/app.jar"]
\ No newline at end of file
+ENTRYPOINT ["java", "-jar", "/app.jar"]
+=======
```
Comment (swevelyne) [inline Dockerfile]: Oops

File: Dockerfile [hunk 3]
```diff
@@ -17,4 +18,34 @@ COPY ${JAR_FILE} app.jar
...
+ENV db_password=$DB_PASSWORD
+ENV AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+ENV AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+
+COPY ${JAR_FILE} app.jar
+WORKDIR /home/docker/data
+#RUN chmod +x /app.jar
+EXPOSE 5000
+ENTRYPOINT ["java", "-jar", "/app.jar"]
+>>>>>>> origin/pre_dev
```
Comment (swevelyne) [inline Dockerfile]: here's another one

**Review by swevelyne — CHANGES_REQUESTED** [review]
Still some small things with Dockerfile, sorry about that

**Pull request closed without merging**